src/client/components/card/status-card02/status-card02.style.css

```diff
@@ -0,0 +1,23 @@
+.check-ph {
+  background: var(--pink);
```

Comment:
we should use the logical names from theme.css instead of color names. like this:
`background: var(--danger-color);`